core/controller/src/main/scala/org/apache/openwhisk/core/controller/Controller.scala

```diff
@@ -176,6 +184,63 @@ class Controller(val instance: ControllerInstanceId,
     LogLimit.config,
     runtimes,
     List(apiV1.basepath()))
+
+  private val controllerCredentials = loadConfigOrThrow[ControllerCredentials](ConfigKeys.controllerCredentials)
+
+  /**
+   * config user memory of ContainerPool
+   */
+  private val configMemory = {
+    implicit val executionContext = actorSystem.dispatcher
+    (path("config" / "memory") & post) {
+      extractCredentials {
+        case Some(BasicHttpCredentials(username, password)) =>
+          if (username == controllerCredentials.username && password == controllerCredentials.password) {
+            entity(as[String]) { memory =>
+              try {
+                val userMemoryMessage = ByteSizeMessage(ByteSize.fromString(memory))
+                if (userMemoryMessage.userMemory.size == 0) {
```

Comment:
I think this is not the only precondition to configure memory.
This value can be smaller than `MemoryLimit.min` then invokers would not be able to run any container.

Reply:
Updated accordingly, the user meomry can't be less than `MemoryLimit.min`